docs/DoctrineOrm.md

```diff
@@ -0,0 +1,131 @@
+# DoctrineOrm component
+
+This component provides methods to integrate Contao and Haste with Doctrine more easily.
+
+## Usage
+
+### Enable relations
+
+#### Using relations in queries
+
+Let's assume the following `Haste-ManyToMany` relation:
+
+```php
+$GLOBALS['TL_DCA']['tl_app_company']['fields']['industries'] = [
+    'exclude' => true,
+    'filter' => true,
+    'inputType' => 'checkbox',
+    'flag' => 1,
+    'foreignKey' => 'tl_app_industry.name',
+    'eval' => ['multiple' => true, 'tl_class' => 'clr'],
+    'relation' => [
```

Comment:
Can't this be inferred from the entity meta data? So that you could do

```
        'type' => 'haste-ManyToMany',
        'entity' => Foobar::class,
        'property' => 'industries'
```

or so?

Reply:
That's an excellent idea, let me find out if it works.

Reply:
Did that in https://github.com/codefog/contao-haste/pull/198/commits/5e3bcd134d20a42b7dcd02df34674fbe770f1264 but maybe we can improve it even more, by adding our own attribute (e.g. `#[HasteManyToMany]`) to the entity property?

Reply:
Yes ;)

Reply:
Well, on second thought I would have to iterate through all entities, pick the one that has the table name we're looking for, then check against attributes… it might affect the performance significantly. This logic should be executed only once and upon the container build or Contao DCA cache build… I am not sure just yet, so let's keep it as is for now.